### ISSUE TITLE
Loading Screen

### ISSUE THREAD

**Issue by hugh5** [UQdeco2800/2022-studio-3#181]
# **Description**
**Task:** [Loading Screen](./181).
**Feature:** [Information Bars](./154)

Loading Screen that appears before main game screen. This screen will show a loading bar which shows that loading progress of the assets required in the main game screen.

# **Dependencies**
List any tasks on which this ticket relies. If there is already an existing ticket, reference it using #xx. If a task needs to be implemented, and the issue does not already exist, make a new ticket and leave it unassigned for someone else to take on. Ensure other team members are aware of the issue.

- [ ] Tutorial/Cut Scene screen must navigate to the loading screen before starting the game (#xx)

# **Milestones**
List of steps that need to be completed for this task.
- [ ]  Loading screen with loading bar implemented (Oct. 1)
- [ ] Wiki Documentation (Oct. 3)
 
**Completion Deadline:** Oct. 3
 
# **Documentation**
- [Wiki](../wiki/Loading-Screen)
 
# **Member**
- Hugh Drummond (@hugh5)
